python/ray/experimental/state.py

```diff
@@ -523,6 +523,9 @@ def task_profiles(self, start=None, end=None, num_tasks=None, fwd=True):
                     del task_info[task_id_hex]
                     heap_size -= 1
 
+        for key, info in task_info.items():
+            self._add_missing_timestamps(info)
```

Comment:
@ericl I moved this function to `task_profiles` because the output of `task_profiles` gets used in multiple places in the UI and so this was easier than fixing all of the different places. Let me know if this change makes sense to you.